assets/src/dashboard/components/cardGrid/index.js

```diff
@@ -59,6 +59,9 @@ const CardGrid = forwardRef(function CardGrid(
       ref={ref}
       role="list"
       data-testid={'dashboard-grid-list'}
+      // TODO: Investigate
+      // See https://github.com/google/web-stories-wp/issues/6671
+      // eslint-disable-next-line styled-components-a11y/no-noninteractive-tabindex
       tabIndex={0}
```

Comment:
This is pretty intentional so that we can access the whole group of stories - there's a few things in this 'card grid' structure that are specific since we have a group that we don't want to create a focus trap with so we need to tab into it and then use the arrow keys to navigate through it (like the story carousel view) and then inside of that we've got some nesting that has buttons within a main button that controls the focus. Anyway, it's rather unique. There's been a mention of updating the designs here and I'm cautiously optimistic that this would mean ditching some of the complexity.  Always down to investigate alternatives!